src/ports/rt-kernel/pnal_snmp.c

```diff
@@ -0,0 +1,103 @@
+/*********************************************************************
+ *        _       _         _
+ *  _ __ | |_  _ | |  __ _ | |__   ___
+ * | '__|| __|(_)| | / _` || '_ \ / __|
+ * | |   | |_  _ | || (_| || |_) |\__ \
+ * |_|    \__|(_)|_| \__,_||_.__/ |___/
+ *
+ * www.rt-labs.com
+ * Copyright 2020 rt-labs AB, Sweden.
+ *
+ * This software is dual-licensed under GPLv3 and a commercial
+ * license. See the file LICENSE.md distributed with this software for
+ * full license information.
+ ********************************************************************/
+
+#include "options.h"
```

Comment:
Please group and sort according to https://rt-labs.com/docs/p-net/implementation_details.html#coding-rules
Use blank lines between groups.

Reply:
It is not possible to follow the coding rules. Header files in the Profinet project do no #include the header files they need. Thus, a c file needs to first #include header files needed by subsequent header files. For example, in order to #include pnet_api.h, one first have to #include options.h.

Reply:
Also see #224

Reply:
Ok, fair.

Reply:
Added some blank lines between groups.